webapp/channels/src/components/admin_console/session_attributes/session_attributes.scss

```
@@ -124,6 +124,8 @@
     font-size: 10px;
     font-weight: 600;
     text-transform: uppercase;
+    user-select: none;
+    cursor: pointer;
```

Stylelint (17.14.0)

[error] Expected "cursor" to come before "user-select" (order/properties-order)

(order/properties-order)